addons/product/models/product_pricelist.py

```diff
@@ -346,7 +346,18 @@ def get_import_templates(self):
             'label': _('Import Template for Pricelists'),
             'template': '/product/static/xls/product_pricelist.xls'
         }]
-    
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        vals_list_nosequence = [vals for vals in vals_list if 'sequence' not in vals]
+        if vals_list_nosequence:
+            last_seq = self.search([], order='sequence desc', limit=1).sequence or 15
+            last_seq = 15 if last_seq < 15 else last_seq
+            for vals in vals_list_nosequence:
+                last_seq += 1
+                vals['sequence'] = last_seq
+        return super().create(vals_list)
```

Comment:
You're overriding the standard `default` logic here (`ir.default` records, `default_` context, ...).
I believe a simple `_default_` method set as `default` on the field would be better, without breaking existing logic.